zwave-property.js

```diff
@@ -82,6 +82,16 @@ class ZWaveProperty extends Property {
     return [motion, motion.toString()];
   }
 
+  parseAccessControlZwValue(zwData) {
+    // zwData for my door sensor can be:
+    // - Clear
+    // - Door/Window Open
+    // - Door/Window Closed
+    // (not sure if this is standard or device specific)
```

Comment:
tried this out on another brand of door sensor which returns the same values, so I could probably look for these exact values instead of the regexp if preferred.